.github/workflows/typescript-build-and-test-package.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: yarn install --production
 
       - name: "build artifact"
-        run: zip -r build.zip build package.json
+        run: zip -r build.zip build package.json node_modules
```

Comment:
why are we adding the `node_modules` folder for an npm package?

Reply:
honestly, why even save an artifact if we are uploading to an npm package? as that is the artifact right? thoughts?

Reply:
We probably don't need them. I don't remember why I added that as I originally had it without.

But the artifact here was just going to be used to publish in another job. But actually, I don't think we need to do that at all.

Reply:
when building a zip for a lambda, the `node_modules` has to be there.  for this, it won't end up in the npm package, and the build artifact isn't really usable either.

Reply:
Alright, I removed all the stuff with creating an artifact. It should now just run the build and test scripts